.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,111 @@
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: ['14.x']
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Install Node dependencies
+      run: |
+        npm ci
+        npm run build --if-present
+        npm install -g jest
+
+    - name: Execute Node integration tests
+      # continue-on-error: true
+      env:
+        NATURAL_LANGUAGE_CLASSIFIER_URL: "https://api.us-south.natural-language-classifier.watson.cloud.ibm.com"
+        NATURAL_LANGUAGE_CLASSIFIER_APIKEY: ${{ secrets.NLC_APIKEY }}
+        NATURAL_LANGUAGE_CLASSIFIER_ID: ${{ secrets.NLC_CLASSIFIER_ID }}
+        ASSISTANT_URL: "https://api.us-south.assistant.watson.cloud.ibm.com"
+        ASSISTANT_APIKEY: ${{ secrets.WA_APIKEY }}
+        ASSISTANT_WORKSPACE_ID: ${{ secrets.WA_WORKSPACE_ID }}
+        ASSISTANT_ASSISTANT_ID: ${{ secrets.WA_ASSISTANT_ID }}
+        COMPARE_COMPLY_URL: "https://api.us-south.compare-comply.watson.cloud.ibm.com"
+        COMPARE_COMPLY_APIKEY: ${{ secrets.CC_APIKEY }}
+        COMPARE_COMPLY_FEEDBACK_ID: ${{ secrets.CC_FEEDBACK_ID }}
+        LANGUAGE_TRANSLATOR_URL: "https://api.us-south.language-translator.watson.cloud.ibm.com"
+        LANGUAGE_TRANSLATOR_APIKEY: ${{ secrets.LT_APIKEY }}
+        NATURAL_LANGUAGE_UNDERSTANDING_URL: "https://api.us-south.natural-language-understanding.watson.cloud.ibm.com"
+        NATURAL_LANGUAGE_UNDERSTANDING_APIKEY: ${{ secrets.NLU_APIKEY }}
+        PERSONALITY_INSIGHTS_URL: "https://api.us-south.personality-insights.watson.cloud.ibm.com"
+        PERSONALITY_INSIGHTS_APIKEY: ${{ secrets.PI_APIKEY }}
+        SPEECH_TO_TEXT_URL: "https://api.us-south.speech-to-text.watson.cloud.ibm.com"
+        SPEECH_TO_TEXT_APIKEY: ${{ secrets.STT_APIKEY }}
+        TEXT_TO_SPEECH_URL: "https://api.us-south.text-to-speech.watson.cloud.ibm.com"
+        TEXT_TO_SPEECH_APIKEY: ${{ secrets.TTS_APIKEY }}
+        TONE_ANALYZER_URL: "https://api.us-south.tone-analyzer.watson.cloud.ibm.com"
+        TONE_ANALYZER_APIKEY: ${{ secrets.TA_APIKEY }}
+        VISUAL_RECOGNITION_URL: "https://api.us-south.visual-recognition.watson.cloud.ibm.com"
+        VISUAL_RECOGNITION_APIKEY: ${{ secrets.VR_APIKEY }}
+        VISUAL_RECOGNITION_COLLECTION_ID: ${{ secrets.VR_COLLECTION_ID }}
+        DISCOVERY_URL: "https://api.us-south.discovery.watson.cloud.ibm.com"
+        DISCOVERY_APIKEY: ${{ secrets.D1_APIKEY }}
+        DISCOVERY_ENVIRONMENT_ID: ${{ secrets.D1_ENVIRONMENT_ID }}
+        DISCOVERY_CONFIGURATION_ID: ${{ secrets.D1_CONFIGURATION_ID }}
+        DISCOVERY_COLLECTION_ID1: ${{ secrets.D1_COLLECTION_ID1 }}
+        DISCOVERY_COLLECTION_ID2: ${{ secrets.D1_COLLECTION_ID2 }}
+        DISCOVERY_COLLECTION_ID3: ${{ secrets.D1_COLLECTION_ID3 }}  # japanese
+
```

Comment:
LGTM, my only question is if Discovery V2 uses its own set of credentials because I was under the impression that it did

Reply:
Your right,  I need to fix this up.  I will do that now and let you know if another review is needed.  Thank you